client/me/purchases/concierge-banner/index.jsx

```diff
@@ -10,35 +10,97 @@ import { localize } from 'i18n-calypso';
  * Internal Dependencies
  */
 import ActionCard from 'components/action-card';
-import { recordTracksEvent } from 'state/analytics/actions';
 import TrackComponentView from 'lib/analytics/track-component-view';
+import conciergeImage from 'assets/images/illustrations/jetpack-concierge.svg';
 
 class ConciergeBanner extends Component {
+	getText() {
+		const { bannerType } = this.props;
+
+		let headerText, mainText, buttonText, buttonHref, illustrationUrl;
+
+		switch ( bannerType ) {
+			case 'upcomingAppointment':
+				headerText = this.props.translate( 'Your appointment is coming up!' );
+				mainText = this.props.translate(
+					'Get ready with your questions for your upcoming Quick Start session appointment.',
+					{
+						comment:
+							"Please extend the translation so that it's clear that these sessions are only available in English.",
+					}
+				);
+				buttonText = this.props.translate( 'Session dashboard' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'View your session dashboard' )` *ES Score: 7*